docs/src/pages/[platform]/connected-components/authenticator/customization/customization.full-ui-customization.android.mdx

```diff
@@ -0,0 +1,38 @@
+### Full UI Customization
+
+In addition to customizing form fields and theming, you can also build a custom UI for one or more of the authenticator steps using a combination of prebuilt widgets from the amplify_authenticator package, and widgets that you build yourself. This can be used for simple use cases, such as adding a logo to the sign in screen, as well as more complex use cases.
```

Comment:
```suggestion
In addition to customizing form fields and theming, you can also build a custom UI for one or more of the authenticator steps using a combination of prebuilt widgets from the `amplify_authenticator` package, and widgets that you build yourself. This can be used for simple use cases, such as adding a logo to the sign in screen, as well as more complex use cases.
```